hikari/api/rest.py

```diff
@@ -25,6 +25,7 @@
 __all__: typing.List[str] = ["ConnectorFactory", "RESTClient"]
 
 import abc
+import builtins
```

Comment:
don't need to import builtins here, just use `int` in the type hint (docstring type still has to be builtins.int though)

Reply:
@davfsa told me to use `builtins.int` there

Reply:
I told you to use it on the docstring, not to actually import it. The pdoc template we have uses `builtins.` to indicate that it has to look in the builtins to link it

Reply:
i did put it there